packages/noya-state/src/reducers/canvasReducer.ts

```diff
@@ -152,7 +152,10 @@ export function canvasReducer(
 
             if (!Layers.isPointsLayer(layer)) break;
 
-            draft.selectedPointLists = { [layer.do_objectID]: [0] };
+            //Selects the first point in the first selected layer
+            layerIds.forEach(function (layer, index) {
```

Comment:
The pervious version only added the first layer to the `selectedPointLists`. So if you had multiple layers selected (copying Sketch behavior here) you could not selected a point that was not part of the first layer.

Reply:
I think this new logic is good. 

- We should probably loop over `layerIndexPaths`, rather than `layerIds`, cause we still want to filter out layers that don't have points (e.g. Text layers). I.e. the line above the `if (!Layers.isPointsLayer(layer)) break;` needs to happen for each layer, not just the first.
- I think we can update the comment to say something about initializing a point list for each selected layer